src/main/drivers/system.c

```diff
@@ -248,15 +248,15 @@ void failureMode(failureMode_e mode)
 
 void initialiseMemorySections(void)
 {
-#ifdef USE_ITCM_RAM
+#ifdef USE_ITCM
     /* Load functions into ITCM RAM */
     extern uint8_t tcm_code_start;
     extern uint8_t tcm_code_end;
     extern uint8_t tcm_code;
     memcpy(&tcm_code_start, &tcm_code, (size_t) (&tcm_code_end - &tcm_code_start));
 #endif
 
-#ifdef USE_CCM_CODE
+#ifdef USE_CCM
```

Comment:
Not sure what to do here - ST calls it 'CCM', 'CCM RAM', and 'CCM MEMORY' interchangeably. But if we stay with `USE_ITCM_RAM` then `USE_CCM_RAM` might make sense here.

Reply:
It definitely shouldnt be "CCM memory" - Core Coupled Memory memory. Sounds a bit daft.
Likewise with ITCM - Instruction Tightly Coupled Memory. Imho these acronyms already give sufficient information about what the type and purpose of the memory is. Adding a _RAM suffix only serves to ambiguate whether or not it is meant for instructions or data. 

As above, im not overly fussed though, as long as both renames end up consistent.